esp/bindings/bindutil.cpp

```diff
@@ -724,3 +724,17 @@ IEspHttpException* createEspHttpException(int code, const char *_msg, const char
 {
     return new CEspHttpException(code, _msg, _httpstatus);
 }
+
+int executeCommand(const char* command, StringBuffer& response)
+{
+    char buffer[128];
+    FILE* fp = popen(command, "r");
```

Comment:
We don't normally use popen for such cases, not sure why not. Does it work on Windows/OSX?

Reply:
It should work for Windows. But, I do not have a Windows/OSX build to verify.

How about switching to our jlib function runExternalCommand()?